clients/admin-ui/src/features/custom-fields/cells.tsx

```diff
@@ -1,14 +1,60 @@
+// eslint-disable react/destructuring-assignment
+import {
+  Button,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  MoreIcon,
+  Text,
+  useDisclosure,
+  WarningIcon,
+} from "@fidesui/react";
 import React from "react";
 import { CellProps } from "react-table";
 
-import { EnableCell, MapCell } from "~/features/common/table/";
-import { FIELD_TYPE_MAP } from "~/features/custom-fields/constants";
+import ConfirmationModal from "~/features/common/ConfirmationModal";
+import Restrict from "~/features/common/Restrict";
+import { EnableCell, MapCell, WrappedCell } from "~/features/common/table/";
+import {
+  FIELD_TYPE_MAP,
+  RESOURCE_TYPE_MAP,
+} from "~/features/custom-fields/constants";
 import { useUpdateCustomFieldDefinitionMutation } from "~/features/plus/plus.slice";
-import { CustomFieldDefinition } from "~/types/api";
+import {
+  CustomFieldDefinition,
+  ResourceTypes,
+  ScopeRegistryEnum,
+} from "~/types/api";
 
+export const ResourceTypeCell = (
+  cellProps: CellProps<typeof RESOURCE_TYPE_MAP, string>
+) => {
+  const mappedValue =
+    // eslint-disable-next-line react/destructuring-assignment
+    RESOURCE_TYPE_MAP.get(cellProps.value as ResourceTypes) ?? cellProps.value;
+  return <WrappedCell {...cellProps} value={mappedValue} />;
+};
+
+// eslint-disable-fu
 export const FieldTypeCell = (
   cellProps: CellProps<typeof FIELD_TYPE_MAP, string>
-) => <MapCell map={FIELD_TYPE_MAP} {...cellProps} />;
+) => {
+  /* eslint-disable */
+  // @ts-ignore
+  const value = cellProps.row.original["allow_list_id"]
```

Comment:
can you add a comment for what's going on here?

Reply:
Sure, good idea